tartufo/commands/scan_folder.py

```diff
@@ -14,19 +14,30 @@
     show_default=True,
     help="Recurse and scan the entire folder",
 )
+@click.option(
+    "--git-check/--no-git-check",
+    is_flag=True,
+    default=True,
+    show_default=True,
+    help="Check if folder is a git repo and show confirmation dialog",
```

Comment:
Since we're defaulting to `True`, do you think it makes more sense to change this help text to talking about disabling the check?

Reply:
I was following the pattern for the other option, but makes sense to change it - pushing a fix shortly.